libraries/AP_Proximity/AP_Proximity_Backend.cpp

```diff
@@ -88,7 +88,7 @@ bool AP_Proximity_Backend::ignore_reading(float pitch, float yaw, float distance
 // returns true if database is ready to be pushed to and all cached data is ready
 bool AP_Proximity_Backend::database_prepare_for_push(Vector3f &current_pos, Matrix3f &body_to_ned)
 {
-#if !APM_BUILD_TYPE(APM_BUILD_AP_Periph)
+#if AC_OAPATHPLANNER_ENABLED && !APM_BUILD_TYPE(APM_BUILD_AP_Periph)
```

Comment:
```suggestion
#if AC_OAPATHPLANNER_ENABLED
```

set `AC_OAPATHPLANNER_ENABLED` to zero in `defaults_periph.h`

Similarly elsewhere

Reply:
Fixed.